### PR TITLE
now supports centerMode

### PR DESCRIPTION
**Pull request by devTeaa**
This PR allows center padding for highlighted carousel, all units are in percentage as it's the easiest implementation

### CODE REVIEW

File: src/cat-carousel.js [hunk 1]
```diff
@@ -71,7 +83,14 @@ export default {
       return this.track === this.maxSlide - 1
     },
     carouselItemStyles () {
-      const width = WIDTH_PAGE / this.itemPerPage
+      let width = WIDTH_PAGE / this.itemPerPage
+
+      if (this.centerMode.enabled) {
+        console.log(this.centerMode)
```
Comment (gilvent) [inline src/cat-carousel.js:89]: please remove console.log

File: README.md [hunk 2]
```diff
@@ -134,6 +144,20 @@ Next navigation:
...
 ```
 
+Center mode carousel:
+
+![](demo/center-mode.png)
+
+```HTML
+<cat-carousel
+    :items="items"
+    :item-per-page="5"
```
Comment (hanssagita) [inline README.md:154]: I think @gilvent right about this one.
I think we can remove this one or ignore this props if centerMode.enable value is true

File: demo/index.html [hunk 3]
```diff
@@ -107,6 +107,7 @@
   <cat-carousel
```
Comment (hanssagita) [inline demo/index.html:107]: Don't replace existing demo cause this demo is used for normal mode
I Recommend you duplicate this code and create new one with center mode under existing demo

Comment (hanssagita) [reply demo/index.html:107]: I need to make sure also the existing production code isn't breaking so I think you need to create new one with your mode

**Comment by devTeaa** [comment]
> This works great, but I think this feature will only be usable if item-per-page is 1 (pls CMIWW if there is use case to use centered and item per page is more than 1). Thus, i believe we need to validate item per page value to 1 when center mode is enabled.
> what do you think @hanssagita , @devTeaa ?

Capture

How about we leave the option open and let the user decide how many items the want to center? @hanssagita @gilvent 

I will fix all of the review comments above

File: demo/index.html (hunk 3 above)
Comment (devTeaa) [reply demo/index.html]: Got it

**Pull request merged**